tests/utils/test_uri.py

```diff
@@ -325,7 +326,7 @@ def test_remove_databricks_profile_info_from_artifact_uri(uri, result):
     ('dbfs:/path/a/b/', 'databricks://', 'dbfs://@databricks/path/a/b/'),
     ('dbfs:/path/a/b/', 'databricks://Profile', 'dbfs://Profile@databricks/path/a/b/'),
     ('dbfs:/path/a/b/', 'databricks://profile/', 'dbfs://profile@databricks/path/a/b/'),
-    ('dbfs:/path/a/b/', 'databricks://scope/key', 'dbfs://scope:key@databricks/path/a/b/'),
+    ('dbfs:/path/a/b/', 'databricks://scope:key', 'dbfs://scope:key@databricks/path/a/b/'),
     ('dbfs:/path/a/b/', 'nondatabricks://profile', 'dbfs:/path/a/b/'),
```

Comment:
can we add the test case `'databricks://scope:key/random_stuff'` here? we will just ignore "random_stuff" right?